samples/SwiftDemoApp/SwiftDemoApp/MasterViewController.swift

```diff
@@ -18,6 +18,7 @@ import UIKit
 
 class MasterViewController: UITableViewController {
   var samples: [Any] = []
+  let cellIdentifier = "Cell"
```

Comment:
I'd prefer this as a property inside `tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath)` since it's only used there.

Reply:
refactored back.